lib/network/modules/LayoutEngine.js

```diff
@@ -213,11 +206,27 @@ class HierarchicalStatus {
         }
       }
     }
+
+    // Added: Compress level values to eliminate gaps (for cyclic)
```

Comment:
Don't put added into comments. I can see what is added in the diff and once merged, it will stay in the code forever.